tools/ogstools/.shed.yml

```diff
@@ -0,0 +1,22 @@
+name: suite_ogs_tools
+owner: ufz
+description: A suite of tools for ogs.
```

Comment:
should we define ogs in a first place?

`A suite of tools for OpenGeoSys (ogs)`